website/screens/components/flex/code/examples/gapOrderGrow.ts

```diff
@@ -0,0 +1,34 @@
+import { DxcInset, DxcFlex } from "@dxc-technology/halstack-react";
+import Placeholder from "./Placeholder";
+
+const code = `() => {
+  return (
+    <DxcInset space="2rem">
+      <DxcFlex gap="2rem">
+        <DxcFlex order={3} grow={1}>
+          <Placeholder height="large" />
```

Comment:
I think we should add `order` value inside each placeholder to make the example more understandable.

Reply:
Agree, added!